client/src/App.js

```diff
@@ -32,9 +68,27 @@ const App = () => {
         <MovieList movies={movieList} />
       </Route>
 
-      <Route path="/movies/:id">
-        <Movie addToSavedList={addToSavedList} />
-      </Route>
+      <Route
+        path="/movies/:id"
+        render={(props) => <Movie {...props} addToSavedList={addToSavedList} />}
+      />
+
+      <Route
+        exact
+        path="/update-movie/:id"
+        render={(props) => <UpdateMovie {...props} onSubmit={onEditSubmit} />}
+      />
+      <Route
+        exact
+        path="/add-movie"
+        render={(props) => (
+          <UpdateMovie
```

Comment:
as numi componenta MovieForm in loc de UpdateMovie, deoarece tu o folosesti in use case-uri diferite

Reply:
da, nu am reusit s-o redenumesc ieri